packages/dashboard-v2/src/components/NavBar/NavBar.js

```diff
@@ -77,9 +78,12 @@ export const NavBar = () => (
               partiallyActive
             />
             <DropdownMenuLink
-              to="/logout"
-              as={Link}
+              onClick={async () => {
+                await accountsService.post("logout");
+                navigate("/auth/login");
```

Comment:
I would probably leave a todo here that says to add some kind of toast message if it fails

Reply:
👍